lib/routes/data/styles.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+/*
+ * Sample Data for Styles Route
+ *
+ * @returns {object} - Object containing sample data to use when rendering samples in the Style Guide
+ */
+
+module.exports = {
+  tableData: {
+    name: 'My Latest',
+    data: {
+      headers: [
+        'content',
+        'last edited',
+        'actions',
+        ' ',
+      ],
+      row: [
```

Comment:
Should be `rows` (or in my previous comments, `body`) and it should be an array of arrays, not an array of objects that each contain one key that's an array.

```javascript
rows: [
  [
    'Conversation API',
    'Tuesday, November 15',
    'Edit Latest Revision',
    'Draft',
  ],
  [
    'Conversation Overview',
    'Tuesday, November 16',
    'Edit Latest Revision',
    'Draft',
  ],
]
```